pytest-selenium/pages/base.py

```diff
@@ -194,3 +193,9 @@ def switch_to_window(self, n):
 
         except WebDriverException:
             self.driver.switch_to_window(self.driver.window_handles[n])
+
+    def open_new_tab(self):
+        """"Open new browser tab."""
+
+        self.driver.execute_script("""window.open("","_blank");""")
```

Comment:
instead of using so many double quotes, wrap it in opposing quotes:

'window.open("", "_blank");'
or
"window.open('', '_blank');"

Reply:
@gregfitch corrected this one with latest push